website/blog/2020-11/_index.md

```diff
@@ -0,0 +1,69 @@
+
+### Load Balancer Events During Shoot Reconciliation ([gardener/gardener#3028](https://github.com/gardener/gardener/pull/3028))
+
+As Gardener is capable of managing thousands of clusters, it is crucial to keep operation efforts at a minimum. This release demonstrates this endeavor by further improving error reporting to the end user. During a shoot’s reconciliation, Gardener creates `Services` of type `LoadBalancer` in the shoot cluster, e.g. for VPN or Nginx-Ingress addon, and waits for a successful creation. However, in the past we experienced that occurring issues from the party creating the load balancer (typically [Cloud-Controller-Manager]( https://kubernetes.io/docs/concepts/architecture/cloud-controller/)) are only exposed in the logs or as events. Gardener now fetches these event messages in propagates them to the shoot status in case of a failure. Users can then often fix the problem themselves, if for example the failure discloses an exhausted quota on the cloud provider.
+
+### KonnectivityTunnel Feature Per Shoot([gardener/gardener#3007](https://github.com/gardener/gardener/pull/3007))
+
+Since release `v1.6` Gardener has been capable of reversing the tunnel direction from the seed to the shoot via the `KonnectivityTunnel` feature gate ([more information]( https://github.com/gardener/gardener/blob/master/docs/usage/reverse-tunnel.md)). With this release we make it possible to control the feature per shoot. We recommend to selectively enable the `KonnectivityTunnel`, as it is still in `alpha` state.
+
+### Reference Protection ([gardener/gardener#2771](https://github.com/gardener/gardener/pull/2771)), 
+([gardener/gardener 1708419)](https://github.com/gardener/gardener/commit/17084191c752c206537b9506b54828f4d723d9b7))
+
+Shoot clusters may refer to external objects, like `Secrets` for specified DNS providers or they have a reference to an audit policy `ConfigMap`. Deleting those objects while any shoot still references them causes sever errors, often only recoverable by an immense amount of manual operations effort. To prevent such scenarios, Gardener now adds a new finalizer `gardener.cloud/reference-protection` to these objects and removes it as soon as the object itself becomes releasable. Due to compatibility reasons, we decided that the handling for audit policy `ConfigMaps` is delivered as an opt-in feature first, so please familiarize yourself with the necessary settings in the Gardener Controller Manager component config if you already plan to enable it.
```

Comment:
> so please familiarize yourself with the necessary settings ....

If there's some helpful material focusing on that, this is a good place to add a reference.